packages/react-devtools-shared/src/hook.js

```diff
@@ -192,18 +194,13 @@ export function installHook(target: any): DevToolsHook | null {
         // but Webpack wraps imports with an object (e.g. _backend_console__WEBPACK_IMPORTED_MODULE_0__)
         // and the object itself will be undefined as well for the reasons mentioned above,
         // so we use try/catch instead.
-        if (
-          appendComponentStack ||
-          breakOnConsoleErrors ||
-          showInlineWarningsAndErrors
-        ) {
```

Comment:
~~Why'd we remove the conditional check?~~

~~This was there before so that we'd avoid patching the console unless we had at least one thing to do (one of these settings was true) b'c patching it does have an unfortunate side effect of changing the code location the browser shows next to console logs so that it's DevTools code _instead of_ the original caller.~~

Nevermind. I forgot that we're _always_ overriding now and either hiding or dimming logs. 😄